.github/workflows/autobuild.yml

```diff
@@ -187,7 +187,7 @@ jobs:
     steps:
       - name:                       Select Xcode version for Mac
         if:                         matrix.config.target_os == 'macos' || matrix.config.target_os == 'ios'
-        uses:                       maxim-lobanov/setup-xcode@v1
+        uses:                       maxim-lobanov/setup-xcode@4aa4176a819ae7c019451acfda0bba67bffc6704
```

Comment:
Are you sure on that commit?  It looks like 6bb546d66e639f9364a232811d05ae31df025e04 for v1 to me.  You've got 1.4.1 there.

Reply:
The [logs on emlynmac's 3.9.0 build](https://github.com/emlynmac/jamulus/runs/7582088663?check_suite_focus=true#step:1:41) show [`4aa4176a819ae7c019451acfda0bba67bffc6704`](https://github.com/maxim-lobanov/setup-xcode/commit/4aa4176a819ae7c019451acfda0bba67bffc6704):
```
commit 4aa4176a819ae7c019451acfda0bba67bffc6704 (HEAD -> master, tag: v1.4.1, tag: v1, origin/master, origin/HEAD)
Author: Maxim Lobanov <maxim-lobanov@github.com>
Date:   Tue Jan 4 10:34:23 2022 +0100
```

6bb546d66e639f9364a232811d05ae31df025e04 is the tag 1.0 for me, which is not equal to v1.

So the status quo seems correct, I think?

Reply:
Ah, they're using v1 for "latest"?  That's not nice :).  Fair enough.